config.go

```diff
@@ -1812,7 +1814,9 @@ func parseRPCParams(cConfig *lncfg.Chain, nodeConfig interface{},
 
 	fmt.Println("Attempting automatic RPC configuration to " + daemonName)
 
-	confFile = filepath.Join(confDir, fmt.Sprintf("%v.conf", confFile))
+	if confFile == "" {
```

Comment:
feels like these checks and replacements should all happen in the same func. I think we should pass `extractBitcoindRPCParams` exactly what it expects. So that means doing all the dataDir, conffile, cookiefile checks here in `parseRPCParams`.

Reply:
That's true, but I think is out of scope for this PR since there's already some checks and replacements of `dataDir` in `extractBitcoindRPCParams`. Perhaps a followup can deal with this.